package.json

```diff
@@ -15,7 +15,7 @@
 	},
 	"license": "SEE LICENSE IN LICENSE.txt",
 	"engines": {
-		"node": ">=14.17.0"
+		"node": "^16.13.0 || >=18.0.0"
```

Comment:
For my knowledge, when is this value read/enforced?

Reply:
It's more of a declaration of which Node versions are supported. Not strictly enforced at runtime. https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines.

Since we are dropping support for Node 14 though, it's a breaking change and we need to call it out in the changelog and a minor version bump (would be a major version bump, but since we are on 0.x, minor is treated as major and can include breaking changes).

Reply:
https://docs.npmjs.com/cli/v9/configuring-npm/package-json#engines

As mentioned in this doc it is advisory only, and will only produce warnings since we don't specify `engine-strict`

Reply:
gotcha, thanks